src/dpp/sslclient.cpp

```diff
@@ -504,6 +505,14 @@ void ssl_client::read_loop()
 							case SSL_ERROR_NONE:
 								/* Data received, add it to the buffer */
 								if (r > 0) {
+									std::string data(server_to_client_buffer, r);
+									std::smatch matches;
+
+									if(std::regex_search(data, matches, std::regex(R"(^(HTTP\/1\.1 .+))"))) {
```

Comment:
won't this regex match the whole document? what if it's a discord message with "200" in the message content?

Reply:
No, it shouldn't. Granted, I used it from #714 but I'll double check this tomorrow.

Reply:
dont use std::regex. It is really really slow, like, unbearably so. You should just use string splitting instead using basic std::string methods.